docker/php/8.1/fpm-alpine/Dockerfile

```diff
@@ -50,4 +51,5 @@ RUN apk del .phpize_deps
 
 RUN ln -s /var/www/ieducar/artisan /usr/local/bin/artisan
 
-RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+ENV COMPOSER_VERSION=2.5.1
```

Comment:
Boa! Consegue mover para o início do arquivo por favor, facilita quem procura quais variáveis podem ser modificadas.

Reply:
Feito @edersoares!

Tinha deixado no final devido ser uma variável que pode ser modificada e nesse caso a compilação poderia ser mais rápido devido o Docker utilizar as camadas anteriores não alteradas.